docs/configuration.asciidoc

```diff
@@ -8,29 +8,32 @@ experimental[]
 Once the driver has been installed, in order for an application to be able to connect to {es} through ODBC, a set of configuration parameters must be provided to the driver. Depending on the application, there are generally three ways of providing these parameters:
 
 * through a connection string;
-* using a User or a System DSN;
+* using a User DSN or System DSN;
 * through a File DSN.
 
 DSN (_data source name_) is a generic name given to the set of parameters an ODBC driver needs to connect to a database.
 
 We will refer to these parameters as _connection parameters_ or _DSN_ (despite some of these parameters configuring some other aspects of a driver's functions; e.g. logging, buffer sizes...).
 
-Using a DSN is the most widely used, simplest and safest way of performing the driver configuration. Constructing a connection string, on the other hand, is the most spartan way and consequently the least common method.
+Using a DSN is the most widely used, simplest and safest way of performing the driver configuration. Constructing a connection string, on the other hand, is the most simplistic way and consequently the least common method.
```

Comment:
I was hinting to the "no comfort" (though not complex) meaning. It's definitely not the most simplistic, though.
Maybe "laborious"?